spec/services/users/delete_account_service_spec.rb

```diff
@@ -134,6 +143,13 @@
         expect(
           audit_record.metadata["account_deletion_notification_sent_at"]
         ).to eq(user.account_deletion_notification_sent_at.iso8601)
+
+        expect(audit_record.metadata["cohort_ids"]).to eq(
+          [student.cohort.id, student_2.cohort.id]
+        )
+        expect(audit_record.metadata["organisation_id"]).to eq(
+          user.organisation&.id
```

Comment:
This should be `organisation.id`. You _know_ in this spec which org this is going to be. There's no need to go via the `user` record.